src/modules/rlm_lua/lua.c

```diff
@@ -471,6 +494,32 @@ static int _lua_list_iterator_init(lua_State *L)
  *
  *
  */
+
+static void rlm_lua_set_radiusd_constants(lua_State *L, char const *name)
+{
+	const radiusd_constants_t *p = radiusd_constants;
+
+	lua_newtable(L);
```

Comment:
Should take the name of the existing API table we use for iterators and other things, and insert into that instead. For robustness it should first check if the table exists, and then if it doesn't create it.